src/mongo/db/startup_warnings_common.cpp

```diff
@@ -197,6 +197,14 @@ void logCommonStartupWarnings(const ServerGlobalParams& serverParams) {
             "warning");
     }
 
+    if (serverParams.rateLimit != 1) {
+        LOGV2_WARNING_OPTIONS(
+            29074,
+            {logv2::LogTag::kStartupWarnings},
+            "Instance configuration contains non-default value for the 'rateLimit' parameter. "
+            "This parameter is deprecated and will be removed in the next major release after 4.4");
```

Comment:
```suggestion
            "This parameter is deprecated by the operationProfiling.slowOpSampleRate parameter, and will be removed in the next major release after 4.4");
```

Reply:
I thought we could point them to the new correct parameter in the same error message.

Reply:
@akira-kurogane 
I updated the message